projects/1.20/assets/tetra/tetra/lang/zh_cn.json

```
@@ -0,0 +1,2465 @@
+  "tetra.stats.speed": "速度",
+  "tetra.stats.speed.tooltip": "影响物品的挥动速度\n \n§7挥动间隔§e%s§7秒\n§7采集效率§e%s§7倍",
+  "tetra.stats.speed.tooltip_extended": "挥动“冷却”由屏幕中央的指示条显示。可以在冷却完毕前挥动，但如此攻击造成的伤害较少。横扫、急速打击等效果只有在攻击指示条完全充满时才会被触发\n \n§7速度是采集效率的倍数，它会加快或减缓方块被破坏的速度。倍数计算方式如下：§f§o倍数 = 速度 * 0.5 + 0.5§r§7",
+  "tetra.stats.draw_speed": "拉弓耗时",
+  "tetra.stats.draw_speed.tooltip": "拉满此物品需要§e%s§r秒",
+  "tetra.stats.draw_speed_normalized": "拉弓耗时",
+  "tetra.stats.draw_speed_normalized.tooltip": "拉弓耗时改变量：§e%s§r秒",
+  "tetra.stats.ability_speed": "冷却",
+  "tetra.stats.ability_speed.tooltip": "此物品能力的基础冷却时间为§e%s§r秒",
+  "tetra.stats.ability_speed_normalized": "冷却",
+  "tetra.stats.ability_speed_normalized.tooltip": "基础冷却时间改变量：§e%s§r秒",
+  "tetra.stats.speed_normalized": "速度",
+  "tetra.stats.speed_normalized.tooltip": "攻击速度改变量：§e%s",
+  "tetra.stats.attack_damage": "伤害",
+  "tetra.stats.attack_damage.tooltip": "此物品攻击会造成§e%s§r点伤害",
+  "tetra.stats.attack_damage.tooltip_extended": "物品能力造成的伤害通常基于物品本身的攻击伤害",
+  "tetra.stats.attack_damage_normalized": "伤害",
+  "tetra.stats.attack_damage_normalized.tooltip": "攻击伤害改变量：§e%s§r点",
+  "tetra.stats.draw_strength": "力量",
+  "tetra.stats.draw_strength.tooltip": "拉满时射出的弹射物会造成§e%s~%s§r点伤害，其初速度为§e%s§r格每刻",
+  "tetra.stats.draw_strength.tooltip_extended": "伤害计算基于原版的弹射物行为。伤害的下限取决于组件与改进，上限计算方法如下：§f§o上限 = 下限 * 1.5 + 1§r§7。由未拉满的弓射出的弹射物伤害基于下限计算，例如：从拉开50%%的弓射出的弹射物会造成50%%伤害下限的伤害",
+  "tetra.stats.power": "力量魔咒",
+  "tetra.stats.power.tooltip": "通过魔咒将弹射物的伤害增加§e%s§r，弹射物速度不受影响",
+  "tetra.stats.quick_charge": "快速装填魔咒",
+  "tetra.stats.quick_charge.tooltip": "通过魔咒将装填时间缩短§e%s§r秒",
+  "tetra.stats.ability_damage": "伤害",
+  "tetra.stats.ability_damage.tooltip": "此物品能力的基础伤害为§e%s§r点",
+  "tetra.stats.bleeding": "流血",
+  "tetra.stats.bleeding.tooltip": "攻击实体时有30%%的概率使其流血2秒，共造成§e%s§r点伤害，对亡灵生物无效",
+  "tetra.stats.backstab": "背刺",
+  "tetra.stats.backstab.tooltip": "从背后击中目标会产生暴击，并造成§e%s§r点额外伤害",
+  "tetra.stats.armor": "护甲",
+  "tetra.stats.armor.tooltip": "在持有时提供§e%s§r点护甲值",
+  "tetra.stats.toughness": "韧性",
+  "tetra.stats.toughness.tooltip": "在持有时提供§e%s§r点盔甲韧性",
+  "tetra.stats.shieldbreaker": "破盾",
+  "tetra.stats.shieldbreaker.tooltip": "攻击举起盾牌的实体时有§e25%%§r概率禁用其盾牌§e5§r秒（或其模块化盾牌基础冷却时间的§e75%%§r）",
+  "tetra.stats.blocking": "格挡",
+  "tetra.stats.blocking.level": "格挡时长",
+  "tetra.stats.blocking.efficiency": "格挡冷却",
+  "tetra.stats.blocking.tooltip": "按住右键可以格挡来袭的攻击和弹射物",
+  "tetra.stats.blocking_duration.tooltip": "按住右键可以格挡来袭的攻击和弹射物，最多持续§e%s§r秒",
+  "tetra.stats.blocking_duration_cooldown.tooltip": "按住右键可以格挡攻击和弹射物，最多持续§e%s§r秒\n \n§7冷却时间：§e%sx§r（§e%s秒§r）",
+  "tetra.stats.blocking.tooltip_extended": "若此物品有第二个能力，则该能力可以通过按下Shift+右击发动，或在按住Shift时松开右键发动（如此操作会使冷却时间延长50%%）",
+  "tetra.stats.blocking_reflect": "反伤",
+  "tetra.stats.blockingReflect.level": "反伤伤害",
+  "tetra.stats.blockingReflect.efficiency": "反伤概率",
+  "tetra.stats.blocking_reflect.tooltip": "§e§7格挡近战伤害时有§e%s§r的概率对攻击者造成§e%s§r点伤害，反伤生效时会触发击中效果",

```

| **审查级别**    |
|:-:|
| ❓疑问 / 需要作者确认 |

**原文: Blocking melee attacks has a §e%s§r chance to deal §e%s§r damage the attacker and apply on hit effects**

```suggestion
  "tetra.stats.blocking_reflect.tooltip": "§e§7格挡近战伤害时有§e%s§r的概率对攻击者造成§e%s§r点伤害，反伤生效时会触发击中效果",
```